src/components/expression/expression-renderer.js

```diff
@@ -968,13 +1118,40 @@ genotet.ExpressionRenderer.prototype.highlightLabelsAfterUpdateData_ =
       this.clickedObject_.column = heatmapData.conditionNames.indexOf(
         this.clickedObject_.conditionName
       );
-      this.signal('expressionClick', this.clickedObject_);
+    }
+    if (this.clickedObject_.row != -1 && this.clickedObject_.column != -1) {
```

Comment:
Up to you. You have roughly eight -1 usages related to undefined rows and columns. Making a constant is not very complex anyway.